drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -1049,7 +1049,17 @@ static int mmcsd_switch(FAR struct mmcsd_state_s *priv, uint32_t arg)
    *  [2:0] Cmd Set
    */
 
+  int ret;
+
+  ret = mmcsd_transferready(priv);
+  if (ret != OK)
+    {
+      ferr("ERROR: Card not ready: %d\n", ret);
+      return ret;
+    }
+
   mmcsd_sendcmdpoll(priv, MMCSD_CMD6, arg);
+  priv->wrbusy = true;
```

Comment:
Does this hold D0 Low while busy, or will it break the interupt wrte complete logic?

Reply:
No, this code sequence is same as other similar place.